#include <limits>

For whatever reason, I've started to get

```stim-src/src/stim/dem/detector_error_model.cc:340:35: error: ‘numeric_limits’ is not a member of ‘std’```  

  If others have had this problem, would it be worth just importing limits to detector_error_model.cc?

--- a/src/stim/dem/detector_error_model.cc
+++ b/src/stim/dem/detector_error_model.cc
@@ -18,6 +18,7 @@
 
 #include <cmath>
 #include <iomanip>
+#include <limits>
 
 #include "stim/simulators/error_analyzer.h"
 #include "stim/str_util.h"

